ofnet.go

```diff
@@ -203,9 +206,16 @@ type ArpModeT string
 const (
 	// ArpFlood - ARP packets will be flooded in this mode
 	ArpFlood ArpModeT = "flood"
-
 	// ArpProxy - ARP packets will be redirected to controller
 	ArpProxy ArpModeT = "proxy"
+
+	// PortType - individual port
+	PortType = "individual"
```

Comment:
These constants were previously defined and I just moved them to a common place. Will probably address it separately